### ISSUE TITLE
Exam sample, question 38

### ISSUE THREAD

**Issue by MatCyg** [Apress/pivotal-certified-pro-spring-dev-exam-02#24]
I believe there is a mistake in question 38. The question is about the following YAML configuration:
```
management:
server:
  port: 9091
```
These are the possible answers:
> What is the effect of this configuration? (choose one)
> A. Actuator endpoints are accessible on port 9091.
> B. The application is accessible on port 9091.

The question indicates there is only one correct answer, but both of them are correct because of the missing whitespace in the YAML snippet. Please correct me if I am wrong

**Comment by ThijsBoehme** [comment]
Indeed, because of the missing whitespace the property affects the entire application. A port for the actuator endpoints is not specified, and thus both server and actuator endpoints are available on port 9091.
